res/database/Default/user/therob.xml

```diff
@@ -1096,71 +1148,82 @@
 						<en>Celebrated final episode for the ladies. Male hip swings. This time the first names of the butt cheek owners have to be guessed. All presented in wonderful landscape shots. You can see the fun the anchorwoman is having.</en>
 						<pl>Uroczysty odcinek finałowy dla pań. Męskie wymachy bioder. Tym razem trzeba odgadnąć imiona właścicieli pośladków. Wszystko przedstawione we wspaniałych ujęciach krajobrazowych. Widać, jak dobrze bawi się prowadząca.</pl>
 					</description>
+					<groups target_groups="134" pro_pressure_groups="0" contra_pressure_groups="0" />
+					<targetgroupattractivity men="0.5" women="1.25" />
+					<data subgenre="300,8" /> <!-- TODO: this one should have no Betty malus if possible -->
```

Comment:
there is no corresponding effect, yet. The flags paid and trash should cause a Betty malus

Reply:
yeah I know, it's a feature request: https://github.com/TVTower/TVTower/issues/1169

Reply:
this instance could be fixed using the broadcast flag 2048

Reply:
applied.